cli/src/main/java/org/opentosca/toscana/cli/commands/transformation/TransformationList.java

```diff
@@ -23,11 +18,14 @@ public TransformationList() {
     }
 
     @Override
-    public void run() {
-        ApiController api = startApi();
+    protected String performCall(ApiController ap, String[] ent) {
+        return null;
+    }
 
+    @Override
+    public void run() {
         try {
-            System.out.println(api.listTransformation(csar));
+            System.out.println(callTransformationList());
```

Comment:
Is this Still WIP or why don't you call the method within the perform call operation?